Dockerfiles/centos6

```diff
@@ -20,4 +22,7 @@ RUN rm -rf $OSCAP_DIR/build/*
 
 WORKDIR /home/$OSCAP_USERNAME/$OSCAP_DIR/build
 
-CMD cmake -DPYTHON_EXECUTABLE=/usr/bin/python2 .. && make -j $BUILD_JOBS && ctest --output-on-failure -j $BUILD_JOBS
+CMD true \
+	&& cmake -DPYTHON_EXECUTABLE=/usr/bin/python2 .. \
+	&& make -j $BUILD_JOBS \
+	&& ctest --output-on-failure -j $BUILD_JOBS
```

Comment:
The style guide isn't explicit about whether or not this should exactly mirror the `RUN` command. I'd be inclined to add the `&& true` here as well, because if any previous command exists with non-zero, `&& true` won't clobber the return code (and won't even get executed), and if everything includes zero return codes, then `&& true` will as well. Might as well keep it consistent everywhere. Same thing in the later Dockerfiles.


Thoughts?

Reply:
Sure, I like your view. I have done that in the rebase. It is not common to have chained commands in `CMD`, but formatting it like this makes it very easy to see what's going on, and diffs also become more useful.